lib/mongoid/scroll/cursor.rb

```diff
@@ -1,17 +1,17 @@
 module Mongoid
   module Scroll
     class Cursor
-      attr_accessor :value, :tiebreak_id, :field_type, :field_name, :direction
+      attr_accessor :value, :tiebreak_id, :field_type, :field_name, :direction, :include_current
 
       def initialize(value = nil, options = {})
         @field_type, @field_name = Mongoid::Scroll::Cursor.extract_field_options(options)
         @direction = options[:direction] || 1
+        @include_current = options[:include_current] || false
         parse(value)
       end
 
       def criteria
         mongo_value = value.class.mongoize(value) if value
-        compare_direction = direction == 1 ? '$gt' : '$lt'
         cursor_criteria = { field_name => { compare_direction => mongo_value } } if mongo_value
```

Comment:
I guess that here `compare_direction` should actually remain `'$gt'` or `'$lt'` even if `include_current` is true. Otherwise we'd include all the records having the same value as the tiebreak.

If is use the example from the readme:
```
{ "$or" : [
  { "position" : { "$gt" : 13 }},
  { "position" : 13, "_id": { "$gt" : ObjectId("511d7c7c3b5552c92400000e") }}
]}
```
what we want with `include_current` is
```
{ "$or" : [
  { "position" : { "$gt" : 13 }},
  { "position" : 13, "_id": { "$gte" : ObjectId("511d7c7c3b5552c92400000e") }}
]}
```
not
```
{ "$or" : [
  { "position" : { "$gte" : 13 }},
  { "position" : 13, "_id": { "$gte" : ObjectId("511d7c7c3b5552c92400000e") }}
]}
```
If you confirm it, I'll add a fix for this case and a spec.